tests/homework18/homework18.spec.js

```diff
@@ -0,0 +1,207 @@
+import { test, expect } from '@playwright/test';
+
+test.describe("Registration Form Tests", () => {
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto("");
+    console.log('Page URL:', page.url());
+    const SignUpButton = page.locator('.hero-descriptor_btn.btn.btn-primary');
+    await SignUpButton.click();
+  })
+
+  test.describe('Positive Scenario', () => { 
+    test.afterEach('Removing created user', async ({ page }) => {
+      const SettingsButton = page.locator('a.btn.btn-white.btn-sidebar.sidebar_btn:has-text("Settings")');
+      await SettingsButton.click();
+      await expect(page).toHaveURL("/panel/settings");
+      await page.click('button.btn.btn-danger-bg:has-text("Remove my account")');
+      await page.click('button.btn.btn-danger:has-text("Remove")');
```

Comment:
why is it so complicated?

you can use hasText
example
` page.locator('x-details', { hasText: 'Details' })`

Reply:
Updated